sdk/src/main/java/io/opentelemetry/sdk/trace/TracerSdk.java

```diff
@@ -59,6 +60,14 @@
 
   private volatile boolean isStopped = false;
 
+  public TracerSdk() {
+    this(Resource.getEmpty());
```

Comment:
is this valid? should we even allow a non-named tracer to exist?

Reply:
I was surprised that `TracerSdk` and its ctor were public in the first place, maybe we should generally refactor this? Currently - apart from test code - only `InMemoryTracing` is accessing the ctor directly.

Reply:
Please create an issue to track this. thanks!

Reply:
done! #650